src/components/molecules/CardInput/CardInput.tsx

```diff
@@ -1,19 +1,7 @@
-import React from "react";
-import { CardElement } from "@stripe/react-stripe-js";
 import "./CardInput.scss";
 
-const CARD_ELEMENT_OPTIONS = {
-  style: {
-    base: {
-      color: "white",
-      fontSmoothing: "antialiased",
-      fontSize: "16px",
-    },
-  },
-};
-
 const CardInput = () => {
-  return <CardElement options={CARD_ELEMENT_OPTIONS} />;
+  return;
 };
```

Comment:
Why is this still here?

Reply:
fixed in 86ea8ad01c98485b02bc0029048517d83d6b9f3d